casper/src/main/scala/coop/rchain/casper/SignDeployment.scala

```diff
@@ -28,10 +30,10 @@ object SignDeployment {
   }
 
   def verify(deployData: DeployData): Option[Boolean] =
-    SignaturesAlg(deployData.sigAlgorithm).map { alg =>
+    SignaturesAlg(deployData.sigAlgorithm) >>= { alg =>
       val toVerify = clear(deployData).toByteString.toByteArray
```

Comment:
Just curious. Why is `clear` here. 

All the integration tests related to the `deploy and propose` failed return with `Failure! Invalid block: DeployNotSigned`.

Reply:
`clear` might be a misleading name; maybe `normalize` is better

The integration tests related to `deploy` `propose` are failing because we aren't signing the deploys, which needs to be done off-chain.

Reply:
So every deploy is signed by empty bytes? It doesn't look right to me.

Reply:
Oh, there is some other data in `DeployData`. My bad.

Reply:
`verify` is used to verify transactions. Maybe you meant to look at

```
  def sign(sec: PrivateKey, deployData: DeployData, alg: SignaturesAlg = Secp256k1): DeployData = {
    val toSign    = clear(deployData).toByteString.toByteArray
    val hash      = Blake2b256.hash(toSign)
    val signature = alg.sign(hash, sec)
    fill(deployData)(alg.toPublic(sec), signature, alg.name)
  } 
```

Reply:
Yeah